### PR TITLE
[Windows] Enable webapi-appsecurity-external-tests suite for windows testing

### PR DESCRIPTION
**Pull request by zhuyongyong**
- Move cordova/cordova-appsecurityapi-android-tests to webapi/webapi-appsecurity-external-tests
- Update 27 cases since the spec update
- Enable webapi-appsecurityapi-external-tests suite for windows testing
- Add 4 cases for "sendRequest" method
- Fail bug: https://github.com/01org/AppSecurityApi/issues/9

Impacted tests(approved): new 4, update 78, delete 0
Unit test platform: Crosswalk Project for Windows 19.48.489.0
Unit test result summary: pass 81, fail 1, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-6437
BUG=https://crosswalk-project.org/jira/browse/XWALK-6325

### CODE REVIEW

**Comment by zhuyongyong** [comment]
@Honry Pls review it

File: webapi/webapi-appsecurityapi-external-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## Introduction
+
+This test suite is for testing cordova app security api specification:
```
Comment (Honry) [inline webapi/webapi-appsecurityapi-external-tests/README.md]: Now, it is not only for cordova app but also for Windows platform.

File: webapi/webapi-appsecurityapi-external-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* https://software.intel.com/en-us/app-security-api/api
+
+## Pre-condition before pack this suite
+
+For Android:
+
+There are extra plugins needed for packing this suite:
+
+Create extra_plugins directory in cordova-appsecurityapi-android-tests
```
Comment (Honry) [inline webapi/webapi-appsecurityapi-external-tests/README.md]: Please update the suite name cordova-appsecurityapi-android-tests

File: webapi/webapi-appsecurityapi-external-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+## Introduction
+
+This test suite is for testing cordova app security api specification:
+* https://software.intel.com/en-us/app-security-api/api
+
+## Pre-condition before pack this suite
+
+For Android:
```
Comment (Honry) [inline webapi/webapi-appsecurityapi-external-tests/README.md]: For Cordova Android

File: webapi/webapi-appsecurityapi-external-tests/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Create extra_plugins directory in cordova-appsecurityapi-android-tests
+
+Git clone https://github.com/01org/AppSecurityApi.git in extra_plugins directory
+
+For Windows:
+
+Copy windows_crosswalk from https://github.com/01org/AppSecurityApi to the current directory
+
+## Pre-condition before launch the app
```
Comment (Honry) [inline webapi/webapi-appsecurityapi-external-tests/README.md]: Better to describe as: "before testing"

File: webapi/webapi-appsecurityapi-external-tests/appsecurityapi/AppSecurityApi_sendRequest_GENERIC_PLAINTEXT.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
```
Comment (Honry) [inline webapi/webapi-appsecurityapi-external-tests/appsecurityapi/AppSecurityApi_sendRequest_GENERIC_PLAINTEXT.html]: 's/2015/2016'

File: webapi/webapi-appsecurityapi-external-tests/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+## Introduction
+
+This test suite is for testing cordova android app and windows app security api specification:
```
Comment (Honry) [inline webapi/webapi-appsecurityapi-external-tests/README.md]: use: "Cordova Android and Windows App Security API specification".

**Comment by Honry** [comment]
Thanks, @zhuyongyong, LGTM

**Pull request merged**